java/adaptersdk-schema-java-interfaces/src/main/java/io/github/fileanalysissuite/adaptersdk/schema/builders/SchemaObjectBuilder.java

```diff
@@ -0,0 +1,83 @@
+/*
+ * Copyright 2022-2023 Micro Focus or one of its affiliates.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *      http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+package io.github.fileanalysissuite.adaptersdk.schema.builders;
+
+import io.github.fileanalysissuite.adaptersdk.schema.model.Field;
+import java.time.Instant;
+import java.util.List;
+import java.util.function.Consumer;
+import java.util.stream.Stream;
+
+public interface SchemaObjectBuilder
+{
+    void clearField(Field field);
+
+    void setBooleanFieldValue(Field field, Boolean value);
+
+    void setBooleanFieldValue(Field field, Boolean... values);
+
+    void setBooleanFieldValue(Field field, List<Boolean> values);
+
+    void addBooleanFieldValue(Field field, Boolean value);
```

Comment:
With the exception of the `List<>` variant, I can't think why we would use the boxed variants instead of the primitive types.  And obviously the same with the auto-generated code and with all the other boxed primitives.

```suggestion
    void setBooleanFieldValue(Field field, boolean value);

    void setBooleanFieldValue(Field field, boolean... values);

    void setBooleanFieldValue(Field field, List<Boolean> values);

    void addBooleanFieldValue(Field field, boolean value);
```